sdk-core/src/main/java/care/data4life/sdk/attachment/ThumbnailService.kt

```diff
@@ -13,14 +13,13 @@
  * applications and/or if you’d like to contribute to the development of the SDK, please
```

Comment:
Yea, we replaced it by a proper solution. The EmptyRecord was more a hack to move along the RxJava call chain without breaking it in case of error. We now have an RxErrorHandler instead.